ted/src/main/java/frc/robot/subsystems/DrivetrainSubsystem.java

```diff
@@ -258,6 +272,10 @@ public SwerveDriveKinematics getSwerveKinematics() {
     return swerveKinematics;
   }
 
+  public void setUseVision(boolean b){
```

Comment:
proper comments and not super sure what b means to the eventual caller of this api, names should be more self explanatory

Reply:
updated